pages/api/login.js

```diff
@@ -9,7 +9,7 @@ const handler = async (req, res) => {
   } catch (e) {
     // eslint-disable-next-line no-console
     console.error(e);
-    return res.status(500).json({ error: "Unexpected error." });
+    //return res.status(500).json({ error: "Unexpected error." });
```

Comment:
Did you look at what e is returning, if this happens to new users every time we should try and fix the error

Reply:
yea its the same error the pops up on the screen.  if you console log it you can see it if it pops up, I forgot I had commented that out, so ill just uncomment it since it shouldn't be commented out.  its a catch so I cant exactly "fix the error", as that's the purpose of the catch.  it should be working fine, but on the event it doesn't it'll just return an error.  I debugged for a while and we should be getting errors anymore, but anyone is welcome to crack at it.